server/routes/taskList.js

```diff
@@ -35,14 +38,15 @@ module.exports = ({ prisonerService, licenceService, audit }) => router => {
       const licenceStatus = getLicenceStatus(licence)
       const allowedTransition = getAllowedTransition(licenceStatus, req.user.role)
       const { statusLabel } = getStatusLabel(licenceStatus, req.user.role)
+      let taskListModel
+      const errorCodes = await caService.getReasonForNotContinuing(bookingId, res.locals.token)
+      const errorObject = errorCodes ? getAllErrorMessages(errorCodes) : null
```

Comment:
probably better to call this what it represents rather than the type = `errorMessages`

Reply:
ok